pysingfel/particlePlacement.py

```diff
@@ -51,7 +51,7 @@ def distribute_particles(particles, beam_focus_radius, jet_radius, gamma): #beam
     if any(item == True for item in checkList):
         distribute_particles(particles, beam_focus_radius, jet_radius)
     # calculate interaction range
-    R_interaction = np.sqrt(beam_focus_radius**2+beam_focus_radius**2+jet_radius**2)*gamma
+    R_interaction = 2*np.sqrt(beam_focus_radius**2+beam_focus_radius**2+jet_radius**2)*gamma
```

Comment:
Doesn't doubling affect your results when gamma < 1?

Reply:
When gamma=1, everything must stick together. So, assume the worst case that the randomly distributed particles are initially at the edge of the focus region, then the interaction range must cover the entire focus region.
I think it does affect the results when gamma<1, i.e., particles can still form a single cluster when gamma<1.

Another implementation would be: assume every particle stick together initially, and increases the distance between the particles as gamma decreases.